frontend/ui/widget/opdsbrowser.lua

```diff
@@ -520,6 +525,13 @@ function OPDSBrowser:downloadFile(item, format, remote_url)
     end
 end
 
+function OPDSBrowser:createNewDownloadDialog(path, buttons)
+    self.download_dialog = ButtonDialogTitle:new{
+        title = _("Choose format to download into:") .. "\n" .. path,
```

Comment:
Your suggested text doesn't lead the user to what the path on the second line of the title is supposed to represent. I know it might seem obvious, but, IMO, some users might not know when first presented with this dialog. How about "Download format into:"?